xml/ha_config_cli.xml

```diff
@@ -1775,13 +1774,13 @@ hostname (string): Hostname
       </para>
 <screen>&prompt.crm.conf;<command>primitive</command> my-rsc ocf:myCorp:myAppl \
     op monitor interval=60 \
-    op monitor interval=61 role=Master</screen>
+    op monitor interval=61 role="Promoted</screen>
      </step>
      <step>
       <para>
        Create the promotable clone resource:
       </para>
-<screen>&prompt.crm.conf;<command>ms</command> ms-rsc my-rsc</screen>
+<screen>&prompt.crm.conf;<command>clone</command> clone-rsc my-rsc</screen>
```

Comment:
Hi @tahliar , I just noticed we are missing `meta promotable=true` here. So basically it should be:

`clone clone-rsc my-rsc meta promotable=true`

Or:
```
clone clone-rsc my-rsc \
    meta promotable=true
```

Reply:
Thanks @gao-yan! I've fixed that in #272.